<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab07_TNR</class>
 <widget class="QMainWindow" name="Lab07_TNR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab07_TNR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="grid1"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>201</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="grid2"/>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="grid3"/>
   </widget>
   <widget class="QComboBox" name="operation_tnr">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>51</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>*</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>-</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>+</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>*num</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>^</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="equalsBtn_tnr">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>160</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="matr1I_tnr">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>340</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="matr1J_tnr">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>340</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="matr1SetBtn_tnr">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Cahnges</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>340</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>j:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>i:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>340</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>j:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>340</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>i:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="matr2J_tnr">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>340</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="matr2SetBtn_tnr">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>300</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Changes</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="matr2I_tnr">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>340</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="coeff_tnr">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="row_tnr">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>300</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="col_tnr">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>340</y>
      <width>48</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>310</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Row:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>350</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Col:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sortRowBtn_tnr">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>300</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort Row</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sortColBtn_tnr">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>340</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort Col</string>
    </property>
   </widget>
   <widget class="QPushButton" name="matr1ResizeBtn_tnr">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Resize Matrix</string>
    </property>
   </widget>
   <widget class="QPushButton" name="matr2ResizeBtn_tnr">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>380</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Resize Matrix</string>
    </property>
   </widget>
   <widget class="QPushButton" name="matr1RandomBtn_tnr">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Fill Random</string>
    </property>
   </widget>
   <widget class="QPushButton" name="matr2RandomBtn_tnr">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>420</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Fill Random</string>
    </property>
   </widget>
   <widget class="QPushButton" name="matr3RandomBtn_tnr">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>380</y>
      <width>191</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Fill Random</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>430</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="outputBtn_tnr">
    <property name="geometry">
     <rect>
      <x>512</x>
      <y>470</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="resulr_tnr">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>460</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="matr1ArMean_tnr">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>480</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Arithmetic mean</string>
    </property>
   </widget>
   <widget class="QPushButton" name="matr1Max_tnr">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>520</y>
      <width>141</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Find max</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getBtn_tnr">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>41</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Get</string>
    </property>
   </widget>
   <widget class="QLabel" name="get_tnr">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
